frontend/src/hooks/use-import-network.tsx

```diff
@@ -56,6 +58,7 @@ export function useImportNetwork() {
           throw new Error("Error: couldn't import network configuration")
         }
       } catch (err: unknown) {
+        console.log(err)
```

Comment:
suggestion(non-blocking): please remove lingering console log if not required. If required, please use console.error(err)